test/test-e2e.sh

```diff
@@ -40,14 +41,22 @@ setup_skaffold() {
 }
 
 create_cluster() {
-  if ! (${KIND} create cluster --name=e2e --image=${NODE_IMAGE}) ; then
+  KIND_CONFIG=""
+  if [ "${HIGH_AVAILABILITY}" = true ] ; then
+    KIND_CONFIG="$PWD/test/kind-ha-config.yaml"
+  fi
+  if ! (${KIND} create cluster --name=e2e --image=${NODE_IMAGE} --config=${KIND_CONFIG}) ; then
     echo "Could not create KinD cluster"
     exit 1
   fi
 }
 
 deploy_metrics_server(){
-  PATH="$PWD/_output:${PATH}" ${SKAFFOLD} run
+  SKAFFOLD_PROFILE=""
+  if [ "${HIGH_AVAILABILITY}" = true ] ; then
+    SKAFFOLD_PROFILE="test-ha"
+  fi
+  PATH="$PWD/_output:${PATH}" ${SKAFFOLD} run -p ${SKAFFOLD_PROFILE}
```

Comment:
when run make test-e2e, error occoured as follow
![image](https://user-images.githubusercontent.com/69306452/129922545-490e8793-b0fe-41b6-9db1-8c97ec3352d5.png)
In this scenario, we still need 
```
  PATH="$PWD/_output:${PATH}" ${SKAFFOLD} run 
```

Reply:
Thank you for point it out, I kind of forgot about this PR and didn't notice that the tests were failing. I should have fixed the parameter expansion in https://github.com/kubernetes-sigs/metrics-server/pull/808/commits/6b169173bb009ed7dc6114773f706a7c3ed6ef8d.